src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.scss

```diff
@@ -48,12 +52,8 @@
     }
 
     .bcs-activity-feed-active-state {
-        bottom: 0;
-        left: 0;
-        margin: 9px 20px;
-        position: absolute;
```

Comment:
I thought we needed this to be absolute?

Reply:
Nope. It only positioned the active-state section  at the top of the activity feed. It actually should allow any errors/warnings to show up above the feed if needed